config.hh

```diff
@@ -763,6 +763,7 @@ DEF_EX( exCantUseIndexDir, "Can't use index directory to store GoldenDict index
 DEF_EX( exCantReadConfigFile, "Can't read the configuration file", exError )
 DEF_EX( exCantWriteConfigFile, "Can't write the configuration file", exError )
 DEF_EX( exMalformedConfigFile, "The configuration file is malformed", exError )
+DEF_EX( exCantCreateDataDir, "Can't create 'goldendict' folder in $XDG_DATA_HOME", exError )
```

Comment:
This line should be moved under the `exCantUseIndexDir` line. And probably should be renamed *and* reworded to be consistent with the two other similar error messages.